core/src/autogluon/core/utils/try_import.py

```diff
@@ -48,9 +48,9 @@ def try_import_mxnet():
 
 def try_import_ray():
     ray_max_version_os_map = dict(
-        Darwin='1.9.0',
+        Darwin='1.11.0',
         Windows='1.8.0',
-        Linux='1.9.0',
+        Linux='1.11.0',
```

Comment:
Why we are upgrading ray. I thought it's pillow and numpy

Reply:
I can remove this. But there were issues with 1.9/1.8 and I made these changes a while back.

Reply:
I see. If it passes CI, then I think it's ok

Reply:
I'm confused, isn't the verison of ray that will be installed still 1.9? This code doesn't change the version of ray that will be installed.

Reply:
Oh you right... this is just for the prompt and import check.